geoportal/c2cgeoportal_geoportal/views/theme.py

```diff
@@ -209,7 +209,7 @@ def build_web_map_service(ogc_server_id):
                     "maxResolutionHint": float("{:0.2f}".format(resolution[1])),
                 }
                 if hasattr(wms_layer, "queryable"):
-                    info["queryable"] = wms_layer.queryable
+                    info["queryable"] = wms_layer.queryable == 1 if hasattr(wms_layer, "queryable") else False
```

Comment:
The code
if hasattr(wms_layer, "queryable") 
is redundant because the same "if" is above the statement

Reply:
Effectively, thanks :-)